site/styles/_meta-detail.scss

```
@@ -590,3 +576,15 @@
     font-size: 12px;
   }
 }
+
+s72-element-switcher {
+  @extend .mb-5;
+}
+
+.meta-detail-infos {
```

Nit: I think should be singular. It doesn't matter the class is used multiple times.

```suggestion
.meta-detail-info {
```

---

- changed to singular